s3.tf

```diff
@@ -36,9 +37,9 @@ resource "aws_s3_bucket" "apiary_data_bucket" {
   acl           = "private"
   request_payer = "BucketOwner"
   policy        = data.template_file.bucket_policy[each.key].rendered
-  tags          = merge(map("Name", each.value["data_bucket"]),
-                        var.apiary_tags,
-                        jsondecode(lookup(each.value, "tags", "{}")))
+  tags = merge(map("Name", each.value["data_bucket"]),
+    var.apiary_tags,
+  jsondecode(lookup(each.value, "tags", "{}")))
```

Comment:
Do you find the new whitespace makes it easier to read the `merge()` args?  To me it's messier.

Reply:
this is from terraform fmt command, my VI editor is setup to run `terraform fmt` on save :-)

Reply:
Wow, hashicorp needs to update `terraform fmt` to not mangle function args that badly. 😃